tests/integration/test_charm.py

```diff
@@ -5,11 +5,12 @@
 import logging
 import pathlib
 import re
-from typing import Dict
+import time
 
 import ops
 import requests
 from boto3 import client
+from conftest import get_mattermost_ip
```

Comment:
Importing from `conftest` usually isn't recommended, should this be in a separate module in the tests directory?

Reply:
Moved to a "utils" module